pkg/handler/processor/guesser/format_guesser.go

```
@@ -37,7 +38,7 @@ var (
 
 func RegisterDocumentFormatGuesser(g DocumentFormatGuesser, name string) {
```

same with this one. Should just return an error.

---

Addressed ✅